analysis/beacon/analyzer.go

```diff
@@ -14,19 +14,19 @@ type (
 	// analyzer implements the bulk of beaconing analysis, creating the scores
 	// for a given set of timestamps and data sizes
 	analyzer struct {
-		connectionThreshold int                                    // the minimum number of connections to be considered a beacon
-		minTime             int64                                  // beginning of the observation period
-		maxTime             int64                                  // ending of the observation period
-		analyzedCallback    func(*dataBeacon.BeaconAnalysisOutput) // called on each analyzed result
-		closedCallback      func()                                 // called when .close() is called and no more calls to analyzedCallback will be made
-		analysisChannel     chan *beaconAnalysisInput              // holds unanalyzed data
-		analysisWg          sync.WaitGroup                         // wait for analysis to finish
+		connectionThreshold int                              // the minimum number of connections to be considered a beacon
+		minTime             int64                            // beginning of the observation period
+		maxTime             int64                            // ending of the observation period
+		analyzedCallback    func(*dataBeacon.AnalysisOutput) // called on each analyzed result
```

Comment:
Golint complained about this variable name because the Beacon.Beacon part of dataBeacon.BeaconAnalysisOutput "stutters."